content/applications/websites/website/pages.rst

```diff
@@ -1,9 +1,187 @@
+:menuselection:`Editor --> Site --> Properties`.
+
+The :guilabel:`Name` tab allows you to:
+
+- rename the page using the :guilabel:`Page Name` field;
+- modify the :guilabel:`Page URL`. In this case, you can :ref:`redirect the old URL to the new one
+  <website/URL-redirection>` if needed. To do so, enable :guilabel:`Redirect Old URL`, then select
+  the :guilabel:`Type` of redirection.
+
+  .. image:: pages/page-redirection.png
+     :alt: Redirect old URL
+
+You can further adapt the page's properties in the :guilabel:`Publish` tab:
+
+- :guilabel:`Show in Top Menu`: Disable if you don't want the page to appear in the menu;
+- :guilabel:`Use as Homepage`: Enable if you want the page to be the homepage of your website;
+- :guilabel:`Indexed`: Disable if you don't want the page to be indexed, i.e., shown in search
+  engine results;
+- :guilabel:`Published`: Enable to publish the page;
+- :guilabel:`Publishing Date`: To publish the page at a specific moment, select the date,
+  click the clock icon to set the time, then click the green check mark to validate your selection.
+- :guilabel:`Visibility`: Select who can access the page:
+
+  - :guilabel:`All`
+  - :guilabel:`Signed In`
+  - :guilabel:`Restricted Group`: Select the group in the :guilabel:`Authorized group` field.
+  - :guilabel:`With Password` : Enter the password in the :guilabel:`Password` field.
+
+.. tip::
+   Alternatively, *some* of these properties can also be modified from :menuselection:`Website -->
+   Site --> Pages`.
+
+Duplicating pages
+~~~~~~~~~~~~~~~~~
+
+To duplicate a page, access the page, then go to :menuselection:`Editor --> Site --> Properties` and
+click :guilabel:`Duplicate Page`. Enter a :guilabel:`Page Name`, then click :guilabel:`OK`. By
+default, the new page is added after the duplicated page in the menu, but you can remove it from the
+menu or change its position using the menu editor.
+
+.. _website/delete-page:
+
+Deleting pages
+~~~~~~~~~~~~~~
+
+To delete a page, proceed as follows:
+
+#. Access the page, then go to :menuselection:`Editor --> Site --> Properties` and click
+   :guilabel:`Delete Page`.
+#. A pop-up window appears on the screen with all links referring to the page you want to delete,
+   organized by category. To ensure website visitors don't land on a 404 error page, you must update
+   all the links on your website referring to the page. To do so, expand a category, e.g.,
+   :guilabel:`Page`, then click on a link to open it in a new window.
+#. Once you have updated the links (or set up a :ref:`redirection <website/URL-redirection>`),
+   select the :guilabel:`I am sure about this` check box, then click :guilabel:`OK`.
+
+.. note::
+   Visitors are redirected to a 404 error page when they try to access a deleted page, either
+   directly or through a link that hasn't been updated. You can prevent this by defining a
+   :ref:`redirection <website/URL-redirection>` for the page's URL.
+
+.. _website/URL-redirection:
+
+URL redirects
```

Comment:
Does this apply to all page types (forum, blog, etc?)
If so, consider moving that part of the documentation to another page or specify this.